addons/medical/functions/fnc_addDamageToUnit.sqf

```diff
@@ -22,8 +23,16 @@
  */
 // #define DEBUG_TESTRESULTS
 
-params [["_unit", objNull, [objNull]], ["_damageToAdd", -1, [0]], ["_bodyPart", "", [""]], ["_typeOfDamage", "", [""]], ["_instigator", objNull, [objNull]], ["_damageSelectionArray", [], [[]]]];
-TRACE_6("addDamageToUnit",_unit,_damageToAdd,_bodyPart,_typeOfDamage,_instigator,_damageSelectionArray);
+params [
+    ["_unit", objNull, [objNull]],
+    ["_damageToAdd", -1, [0]],
+    ["_bodyPart", "", [""]],
+    ["_typeOfDamage", "", [""]],
+    ["_instigator", objNull, [objNull]],
+    ["_damageSelectionArray", [], [[]]],
+    ["_overrideInvuln", true, [true]]
+];
+TRACE_6("addDamageToUnit",_unit,_damageToAdd,_bodyPart,_typeOfDamage,_instigator,_damageSelectionArray,_overrideInvuln);
```

Comment:
```suggestion
TRACE_7("addDamageToUnit",_unit,_damageToAdd,_bodyPart,_typeOfDamage,_instigator,_damageSelectionArray,_overrideInvuln);
```

Reply:
Why do you need to embarrass me like that?